decimal_test.go

```diff
@@ -402,6 +484,21 @@ func TestDecimal_Overflow(t *testing.T) {
 	}
 }
 
+func TestIntPart(t *testing.T) {
+	for _, testCase := range []struct {
+		Dec     string
+		IntPart int64
+	}{{"0.01", 0}, {"12.1", 12}, {"9999.999", 9999}, {"-32768.01234", -32768}} {
```

Comment:
could you please put each of these testcases on a separate line? It's hard to follow otherwise.